db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema.define(version: 2018_10_16_134245) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
This was changed when I ran a `db:migrate`. I'm curious if this is required because we don't have a migration which enables it.

In addition it's enabled by default on all Heroku instances - https://devcenter.heroku.com/changelog-items/425

Reply:
Which postgres version are you running?

Reply:
postgres (PostgreSQL) 10.5 

When you run your migrations does it update this value? Perhaps my version isn't correct

Reply:
Reverted that change 👍 
https://github.com/brave-intl/publishers/pull/1345/commits/cc84aeefd4661eced8eb89faa5eebe048eb71bae